apps/playground/src/app/components/angular-tree-component/angular-tree-component/angular-tree-component.component.html

```diff
@@ -49,7 +49,7 @@ <h2>Checkboxes and context menus</h2>
     class="sky-padding-even-default"
   >
     <sky-angular-tree-wrapper>
-      <tree-root [nodes]="nodes" [options]="optionsNoncascading">
```

Comment:
are we sure this is a typo?

Reply:
The issue here was the casing. I modified both the HTML and TypeScript to upper-case the "C"